backend/app/config.py

```diff
@@ -10,6 +10,7 @@
 METRICS_FILENAME = "metrics.json"
 LOGS_FILENAME = "logs.txt"
 INTERVAL_5SEC = 5
+INTERVAL_1MIN = 60
```

Comment:
I don't see much of a reason of naming constants like this. We should either rename it to what it is actually used for or simply use the values in the code.